lib/api/admin/admin.test.ts

```diff
@@ -0,0 +1,37 @@
+import { initDB } from "../../db/init";
+import {getLockers, getUsers, getRegistrations, deregisterAll} from "./admin";
+
+describe("Testing Admin Functions", () => {
+    it("returns true", () => {
```

Comment:
Given that `initDB()` isn't part of `./admin`, this title does not describe this test in context